Sources/StreamChat/Database/DTOs/UserDTO_Tests.swift

```diff
@@ -43,6 +44,8 @@ class UserDTO_Tests: XCTestCase {
             Assert.willBeEqual(payload.createdAt, loadedUserDTO?.userCreatedAt)
             Assert.willBeEqual(payload.updatedAt, loadedUserDTO?.userUpdatedAt)
             Assert.willBeEqual(payload.lastActiveAt, loadedUserDTO?.lastActivityAt)
+            Assert.willBeEqual(payload.teams.first, loadedUserDTO?.teams?.first?.id)
+            Assert.willBeEqual(loadedUserDTO?.teams?.first?.users.first, loadedUserDTO)
```

Comment:
What exactly are you testing here? 🤯 This is already taken care of by CoreData, we don't need to test it explicitly.